fire/api/model/punkttyper.py

```diff
@@ -420,6 +476,233 @@ def fejlmeldt(self, value: Boolean):
             self._fejlmeldt = Boolean.FALSE
 
 
+class Tidsserie(FikspunktregisterObjekt):
+    __tablename__ = "tidsserie"
+    sagseventfraid = Column(String, ForeignKey("sagsevent.id"), nullable=False)
+    sagsevent = relationship(
+        "Sagsevent", foreign_keys=[sagseventfraid], back_populates="tidsserier"
+    )
+    sagseventtilid = Column(String, ForeignKey("sagsevent.id"), nullable=True)
+    slettet = relationship(
+        "Sagsevent",
+        foreign_keys=[sagseventtilid],
+        back_populates="tidsserier_slettede",
+    )
+
+    punktid = Column(String(36), ForeignKey("punkt.id"))
+    punkt = relationship("Punkt")
+
+    punktgruppeid = Column(Integer, ForeignKey("punktgruppe.objektid"))
+    punktgruppe = relationship("PunktGruppe")
+
+    jessenkoordinatid = Column(Integer, ForeignKey("koordinat.objektid"))
+    jessenkoordinat = relationship("Koordinat")
+
+    referenceramme = Column(String, nullable=False)
+    sridid = Column(Integer, ForeignKey("sridtype.sridid"), nullable=False)
+    srid = relationship("Srid", lazy="joined")
+
+    koordinater = relationship(
+        "Koordinat", secondary=tidsserie_koordinat, back_populates="tidsserier"
+    )
+
+    @reconstructor
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._x0 = None
```

Comment:
Hvorfor er der ikke et _y0 og _z0 ?

(Vi skal måske lige gennemgå funktionaliteten sammen, så jeg kan forstå den.)

Reply:
En forglemmelse. Jeg testede af om der var en fidus i at kunne vælge et vilkårligt `x0` at normalisere tidsserien ud fra. Jeg fik det ikke rigtigt til at virke, det var nok for smart. Ideen var at jeg ville gøre det muligt at sætte to tidsserier i forlængelse af hinanden ved at justere offset på denne måde. Det skal der tænkes lidt mere over, så bare se bort fra det her for nu.